egui_demo_lib/src/apps/demo/plot_demo.rs

```diff
@@ -95,7 +95,19 @@ impl PlotDemo {
             });
         });
 
-        ui.label("Drag to pan, ctrl + scroll to zoom. Double-click to reset view.");
+        ui.label(
+            "Pan by dragging, or with mouse wheel (+ shift = horiz.), or with two-finger swipe.",
+        );
+        if cfg!(target_arch = "wasm32") {
+            ui.label(
+                "Zoom with ctrl + mouse wheel (or ⌘ + wheel on a Mac), or with two-finger pinch.",
+            );
+        } else if cfg!(target_os = "macos") {
+            ui.label("Zoom with ⌘ + mouse wheel, or ⌘ + two-finger swipe.");
+        } else {
+            ui.label("Zoom with ctrl + mouse wheel, or ctrl + two-finger swipe (your experience may vary).");
```

Comment:
I think just `Zoom with ctrl/⌘ + scroll, or with pinch gesture.` works. Two finger swiping on a touch screen or touch pad is often referred to as scrolling.

Reply:
Does the pinch gesture work for you when running natively on MacOS? For me, pinching exclusively works in the browser.
If you like my latest commit (which I committed in parallel to your latest remark), you are welcome to release the PR. Otherwise, I will be happy to get rid of the term "two-finger swipe".

Reply:
<https://github.com/emilk/egui/pull/402/commits/b185d64a19a88add3c2fbc029acd6cc239d4b284> removes the clumsy term "two-finger swipe" as suggested.